frontend/app/routes/protected/multi-channel/sin-confirmation.tsx

```diff
@@ -70,16 +73,37 @@ export default function SinConfirmation({ loaderData, actionData, params }: Rout
   const fetcherKey = useId();
   const fetcher = useFetcher<Info['actionData']>({ key: fetcherKey });
   const isSubmitting = fetcher.state !== 'idle';
-  const contentRef = useRef<HTMLDivElement>(null);
   const recordDetails = loaderData.recordDetails;
   const { dateEn, dateFr } = recordDetails.date;
 
+  const [isPrinting, setIsPrinting] = useState(false);
+  const [sinNumber, setSinNumber] = useState<string>();
+
+  useEffect(() => {
+    if (fetcher.data && isPrinting) {
+      setSinNumber(fetcher.data.sin);
+    }
+  }, [fetcher.data, isPrinting]);
+
+  useEffect(() => {
+    if (sinNumber && isPrinting) {
+      print();
```

Comment:
```suggestion
      window.print(); // or globalThis.print()
```

I think it's better to be explicit about what scope the `print()` function lives in. Feel free to mark this comment as resolved if you want (it's not that important).